### DISCUSSION TITLE
リリースノートの更新を気付きやすくしたい

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#889] [要望・アイデア・不具合]
https://qiita.com/release-notes はいろいろと興味深いアップデートが載っていますが、自分から見に行かないとアップデートに気づけません。
このページにRSSフィードがあればFeedlyのようなRSSリーダーで随時更新をキャッチできるので便利だと思いました。

更新に気づきやすくなるのであれば、RSSフィードにこだわりはないです。
たとえば、

- リリースノートをフォローすると通知欄（画面右上のベルマーク）に新規投稿が通知される
- 記事表示中のサイドバーに今の「コラム」と同じような形式で表示される
- 毎週水曜に送信されるランキングメールに「今週のアップデート」みたいな欄を作って新機能を紹介する

みたいな仕組みでも構いません。

ご検討よろしくお願いします。

**Comment by tanaka-asami415** [comment]
@JunichiIto 
ありがとうございます！
実は、リリースノートの更新も通知欄でお知らせできるようにする予定です💪✨
（実装は少し先になってしまいそうですが…）

他にいただいたアイディアも上記とは別に検討させていただきます！

**Comment by ohakutsu** [comment]
@JunichiIto 
おまたせしました！

通知欄でお知らせできるようにするのは、まだ時間がかかってしまうため
リリースノートのRSSフィードを公開しました :rocket: 
https://qiita.com/release-notes/2024-10-29

ぜひご活用ください！

**Reply by JunichiIto** [reply]
ありがとうございます！さっそくRSSフィードをフォローしました 😄
